examples/gnn_fraud_detection_pipeline/run.py

```diff
@@ -16,6 +16,10 @@
 import os
 
 import click
+# pylint: disable=no-name-in-module
+from stages.classification_stage import ClassificationStage
+from stages.graph_construction_stage import FraudGraphConstructionStage
+from stages.graph_sage_stage import GraphSAGEStage
```

Comment:
Do you know what triggered this change? Typically this means `isort` is misconfigured, because these imports should be down at the bottom. 

Maybe try something like:
```suggestion
# pylint: disable=no-name-in-module
from .stages.classification_stage import ClassificationStage
from .stages.graph_construction_stage import FraudGraphConstructionStage
from .stages.graph_sage_stage import GraphSAGEStage
```

Reply:
Yeah, this def needs some investigation. The changes are from running `pre-commit run -a`. CI checks stage was also failing, although this change didn't do anything with examples directly.